Data/Scripts/GardenConquest/Core/CommandProcessor.cs

```diff
@@ -21,20 +21,72 @@ class CommandProcessor {
 
 		private static Logger s_Logger = null;
 
-		// TODO: rest of this
-		private static string s_AboutText =
-			"Welcome to Garden Conquest\n\n" +
-			"In GC each grid must have a Hull Classifier ... MORE";
-
 		private static string s_HelpText =
-			"Chat commands:\n\n" +
-			"/gc about - Show basic mod info\n" +
+			"Garden Conquest is a new, open source, Conquest-type mod for " +
+			"Space Engineers that introduces area-control dynamics for " +
+			"combat-focused survival servers.\n\n" +
+			"If you're new to GC, check out the help subtopics below.\n\n" +
+			"Chat Commands:\n" +
 			"/gc help - Show this screen\n" +
-			"/gc help [category] - Show help for a category.  Categories are:\n" +
-			"    limits - Information on grid class limitations\n";
+			"/gc help [category] - Show help for a category.  Categories:\n" +
+			"        classes     - Ship Classes and their limits\n" +
+			"        classifiers - Hull Classifier blocks\n" +
+			"        cps            - Control Points\n" +
+			"        licenses    - Ship License components\n" +
+			"/gc fleet - Information on your faction's fleet \n" +
+			"/gc fleet remove \"Ship Name\"- Disown a ship";
```

Comment:
Probably an even more common use case would be when your ship gets wrecked in battle, is flying off at a million m/s and no one catches it to get your hull classifier, but it's still counting against your limits.